### ISSUE TITLE
Using dask_ml mean_square_error creates concurrent.

### ISSUE THREAD

**Issue by hayesgb** [coiled/dask-xgboost-nyctaxi#7]
The initial example of this work used `sklearn's` implementation of a `metric` to calculate the average error metric for a single trial.  This works, but as the size of the data grows, `computing` the result increases the variance in trial runtimes.

In our original submission here, we use the `dask_ml.metrics` implementation of `mean_squared_error` to optimize performance by leaving computation on the cluster, with:
```
from dask_ml.metrics import mean_squared_error as lazy_mse
from distributed import wait

...

score = lazy_mse(y_test.to_dask_array(lengths=True).reshape(-1,),  predictions.to_dask_array(lengths=True), squared=False)
wait(score)
```

Unfortunately this causes `concurrent.futures._base.CancelledError`, which seems to be related to [dask/distributed #4612](https://github.com/dask/distributed/issues/4612).

Explicitly calling `wait(score)` resolves this, but sporadically triggers `asyncio.exception.CancelledErrors` when running in a Jupyter Notebook.

**Comment by ncclementi** [comment]
I'm going to close this issue as currently the notebook is running.
